ca-certificates/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("USAGE: ca-certificates <url>")
+		os.Exit(1)
+	}
+	_, err := http.Head(os.Args[1])
+	if err != nil {
+		fmt.Println("ERROR:", err)
+		os.Exit(2)
+	}
+	fmt.Println("SUCCESS!")
+}
```

Comment:
Would it make sense to orient this around an HTTP request/response on `$PORT` that returns `200 OK` or `4xx`, to be consistent with most of the other samples?

Reply:
I'm not opposed but curious what you see as the main advantage in changing this. It wasn't necessarily the intention behind adding this sample but it doesn't seem bad to have an example that accepts arguments and runs a CLI task to completion, given that some images built with Paketo are intended to be used that way.

Reply:
My motivation is largely selfish (though I've been known to raise breakages here as they happen).

I am deploying all of these samples to knative downstream, so the more of these I can run, the more validation I get.  I've excluded this for now (as with other run-to-completion things like `ruby/rake`).

Absolutely no pressure, just what put this on my radar 😇